src/main/java/frc/robot/commands/intake/RunIntakeCommand.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class RunIntakeCommand extends Command
+ {
+  /** Creates a new RunIntakeCommand. */
+
+  private IntakeSubsystem m_intakeSubsystem;
+  double m_Percent;
+
+  public RunIntakeCommand(IntakeSubsystem intake, double percent)
+  {
+    this.m_intakeSubsystem = intake;
+    this.m_Percent = percent;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(intake);
```

Comment:
You have several commands here the require the intake. If you attempt to run multiple commands, such as adjusting the wrist angle and shoot at the same time they will collide and one will interrupt the other. For the intake, the subsystems shouldn't be required in any of the commands as they will need to run concurrently.